src/java/fr/paris/lutece/plugins/forms/service/FormService.java

```diff
@@ -465,16 +439,20 @@ private boolean canUserAccessFile( HttpServletRequest request, int nIdForm )
      *            The form
      * @param strUserGuid
      *            The user guid
+     * @param formResponseManager
+     *
      * @return the created {@code FormResponseManager} object
      */
-    public FormResponseManager createFormResponseManagerFromBackUp( Form form, String strUserGuid )
+    public FormResponseManager createFormResponseManagerFromBackUp( Form form, String strUserGuid, FormResponseManager formResponseManager )
```

Comment:
Have you conducted an assessment of potential regressions resulting from the modification of method signatures?